Eldev

```diff
@@ -0,0 +1,11 @@
+; -*- mode: emacs-lisp; lexical-binding: t; no-byte-compile: t -*-
+
+;; Autodetermined by `eldev init'.
+(eldev-use-package-archive 'melpa)
+
+(eldev-use-plugin 'autoloads)
+(eldev-use-plugin 'undercover)
```

Comment:
We might want to send statistics to coveralls or something with this. Related docs: https://github.com/doublep/eldev#undercover-plugin

Reply:
Sounds good. That can probably be handled in a new PR.